gadopt/stokes_integrators.py

```diff
@@ -11,7 +11,8 @@
 import firedrake as fd
 
 from .approximations import BaseApproximation, AnelasticLiquidApproximation
-from .momentum_equation import StokesEquations
+from .equations import Equation
+from .momentum_equation import residual_terms_stokes as terms_stokes
```

Comment:
If you have to rename "residual terms" to just "terms" either:
- it's redundant to include "residual", and it should just be `terms_stokes`;
- don't rename the import here.

Reply:
Makes sense, thank you.